storage/client_test.go

```diff
@@ -496,20 +496,20 @@ func (m *multiTestContext) RangeLookup(
 	return m.distSenders[0].RangeLookup(key, desc, considerIntents, useReverseScan)
 }
 
-func (m *multiTestContext) makeContext(i int) storage.StoreContext {
-	var ctx storage.StoreContext
+func (m *multiTestContext) makeStoreConfig(i int) storage.StoreConfig {
+	var cfg storage.StoreConfig
 	if m.storeContext != nil {
-		ctx = *m.storeContext
+		cfg = *m.storeContext
```

Comment:
This is confusing. Why not rename `multiTestContext.storeContext` to `multiTestContext.storeConfig`?